app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,287 @@
+@import 'bootstrap';
+
+$white: #ffffff;
+$black: #000000;
+$yellow: #feb600;
+$grey: #808080;
+$darkGrey: #a9a9a9;
+$dimGrey: #696969;
+$red: #ff0000;
+$lightYellow: #ffd700;
+
+html, body {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  background-color: $white;
+  height: 100%;
+  width: 100%;
+  font-family: Verdana, sans-serif;
+}
+
+.login {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  background: url(commit.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  p.alert {
+    color: $white;
+    font-weight: bold;
+    text-shadow: 0px 0px 10px #000;
+  }
+
+  div.container {
```

Comment:
If you do this, it will affect the whole `.container` in all files. What is better is to just give your tag a new class and apply these CSS to that class.

Basically, it is not advised to apply CSS directly into Bootstrap classes.

Reply:
Oh okay I see that all these is enclosed in `.login`.

I still advise you to just give all these tags that you are applying to, a new class. For example, the `.container` inside your `.login` can be given the class `.login-container`, then apply to that specific class. Imagine if you later want to add a new `.container` in your HTML files - now suddenly the new container also has this styles, and if you want to separate them you need to do this anyways. Better do it now, and save headache later. :)